teachertool/src/style.overrides/Button.scss

```diff
@@ -39,4 +39,9 @@
 
 .common-button.disabled {
     opacity: 0.8;
+}
+
+.common-input-group>.common-button {

```

Comment:
I don't think we want this styling applied to all instances of .common-input-group>.common-button. This is a special case, so it should go in ShareLinkInput.module.scss, as `button[class~="common-button"]`.